scalecodec/type_registry/polymesh.json

```diff
@@ -92,12 +92,35 @@
         ["total_supply", "Balance"],
         ["owner_did", "IdentityId"],
         ["divisible", "bool"],
-        ["asset_type", "AssetType"],
-        ["primary_issuance_agent", "Option<IdentityId>"]
+        ["asset_type", "AssetType"]
       ]
     },
     "PalletName": "String",
     "DispatchableName": "String",
+    "AssetPermissions": {
+      "type": "enum",
+      "type_mapping": [
+        ["Whole", "Null"],
+        ["These", "Vec<Ticker>"],
+        ["Except", "Vec<Ticker>"]
+      ]
+    },
+    "PortfolioPermissions": {
+      "type": "enum",
+      "type_mapping": [
+        ["Whole", "Null"],
+        ["These", "Vec<PortfolioId>"],
+        ["Except", "Vec<PortfolioId>"]
+      ]
+    },
+    "DispatchableNames": {
+      "type": "enum",
+      "type_mapping": [
+        ["Whole", "Null"],
+        ["These", "Vec<DispatchableName>"],
+        ["Except", "Vec<DispatchableName>"]
+      ]
```

Comment:
This should already existed IMO? I am not missing something

Reply:
No, this is part of the external agents changes